tests/.infra/crw-ci/pr-check/k8s/Jenkinsfile

```diff
@@ -269,6 +289,11 @@ EOL"""
                             fi
                             done
                     """
+
+                    echo "CheCluster yaml content"
```

Comment:
This step looks redundant so as pipeline is storing checluster content at cleanup stage

Reply:
You're right. Fixed.